src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -1296,6 +1295,25 @@ internal ReplicationMode ReplicationMode
 
         string? _options;
 
+        /// <summary>
+        /// Configure the way arrays of value types are returned when requested as object instances.
+        /// </summary>
+        [Category("Advanced")]
+        [Description("Configure the way arrays of value types are returned when requested as object instances.")]
+        [DisplayName("ValueTypeArrayNullability")]
+        [NpgsqlConnectionStringProperty]
+        public ValueTypeArrayNullability ValueTypeArrayNullability
```

Comment:
Maybe just ArrayNullability for brevity? Or ArrayNullabilityMode? The fact that it's only for value types can be implicit...